packages/logs/src/domain/trackNetworkError.ts

```diff
@@ -46,6 +59,37 @@ export function trackNetworkError(configuration: LogsConfiguration, errorObserva
   }
 }
 
+// TODO: ideally, computeXhrResponseData should always return a string instead of `any`. But to keep
+// backward compatibility, in the case of XHR with a `responseType` different than "text", the
+// response data should be whatever `xhr.response` is. This is a bit confusing as Logs event 'stack'
+// is expected to be a string. This should be changed in a future major version as it could be a
+// breaking change.
+export function computeXhrResponseData(xhr: XMLHttpRequest, configuration: LogsConfiguration): any {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏even if `xhr.response` type is `any`, what about handling it as `unknown` on our side? or reflect what we expect with something like `string | object`? 

and having the same type for `onResponseDataAvailable(responseData: any) {`

Reply:
I went with `unknown` because in reality it can be a variety of things: https://developer.mozilla.org/fr/docs/Web/API/XMLHttpRequest/response